Mathlib/Analysis/InnerProductSpace/Spectrum.lean

```diff
@@ -7,6 +7,8 @@ import Mathlib.Analysis.InnerProductSpace.Rayleigh
 import Mathlib.Analysis.InnerProductSpace.PiL2
 import Mathlib.Algebra.DirectSum.Decomposition
 import Mathlib.LinearAlgebra.Eigenspace.Minpoly
+import Mathlib.LinearAlgebra.Semisimple
+import Mathlib.Analysis.InnerProductSpace.Projection
```

Comment:
This suggests to me that this is the wrong file for that `Semisimple` result‌. I think it should be possible to put it in `Mathlib.Analysis.InnerProductSpace.Projection` with `Mathlib.LinearAlgebra.Semisimple` as the only import there.

Reply:
Ok cool. Will do that. This is a nice way to spot such a thing, too...